TreeDB/collections/vector_index_column_graph.go

```
@@ -38,12 +39,239 @@ func (unsupportedColumnVectorGraphIndexLoader) LoadColumnVectorGraphIndex(Column
+		Vectors:         state.vectors,
+		InvNorms:        state.invNorms,
+		NeighborOffsets: state.offsets,
+		Neighbors:       state.neighbors,
+		Dimensions:      input.Definition.Dimensions,
+		EntryPoint:      0,
+		EfSearch:        input.Definition.EfSearch,
+	})
+	if err != nil {
+		status = columnGraphPhysicalUnavailableLoadStatus(vectorIndexFallbackColumnGraphInvalid)
+		status.BytesDisk = diag.PhysicalBytesScanned
+		return ColumnVectorGraphIndexLoadResult{Status: status}, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 119) but it returns nil

(nilerr)